app/assets/javascripts/thyme/display_manager.js

```diff
@@ -38,48 +38,50 @@ class DisplayManager {
 
   // Check screen size and trigger the right method
   updateControlBarType() {
+    const dm = this;
+
```

Comment:
I've reverted this in 759d99c368dd9, why did you change it back? I don't think we need to rename `this` since `this` is very common in many object-oriented programming languages (e.g. Java/C++, in Rust/Python the keyword is called `self`). It would only confuse to refer to `this` under another name.

Reply:
The resize()-function didn't work properly anymore. The problem was that in the scope of the listeners down below `this` refered to the element wo which the listener was attached to and not to the display manager. Using a different reference for the current display manager solves this problem.

Reply:
Note that I updated the `updateControlBarType()` function in 8cd43416be24 to reduce its complexity.